lib/phlex/csv.rb

```diff
@@ -113,8 +113,8 @@ def call(buffer = +"", context: nil, delimiter: self.delimiter)
 		buffer
 	end
 
-	def around_row(item)
-		row_template(item)
+	def around_row(...)
+		row_template(...)
```

Comment:
This looks good, but the tests are failing.

Reply:
Oh weird, I'll take a look. They're passing locally